web/src/routes/admin/system-settings/+page.svelte

```diff
@@ -110,6 +111,13 @@
       >
         <FFmpegSettings disabled={$featureFlags.configFile} ffmpegConfig={configs.ffmpeg} />
       </SettingAccordion>
+
+      <SettingAccordion
```

Comment:
Can you place it in the first position (settings are sorted in alphabetical order)?

Reply:
Yes I can!